src/agent/coverage/src/record.rs

```diff
@@ -121,31 +121,34 @@ impl CoverageRecorder {
     #[cfg(target_os = "windows")]
     pub fn record(self) -> Result<Recorded> {
         use debugger::Debugger;
+        use process_control::{ChildExt, Control};
         use windows::WindowsRecorder;
 
         let loader = self.loader.clone();
+        let mut recorder =
+            WindowsRecorder::new(&loader, self.module_allowlist, self.cache.as_ref());
+        let (mut dbg, child) = Debugger::init(self.cmd, &mut recorder)?;
+        dbg.run(&mut recorder)?;
```

Comment:
`run` here won't terminate until the target has exited. Is there a way for us to put the time limit on the child before we call `run`?